scripts/resolve_suggestions_urls.py

```diff
@@ -0,0 +1,54 @@
+import json
+from jsonpointer import resolve_pointer, set_pointer
+
+
+def find_pointers_containing(input_data, search_key, pointer=None):
+    if isinstance(input_data, dict):
+        if pointer and search_key in input_data:
+            yield pointer
+        for k, v in input_data.items():
+            yield from find_pointers_containing(
+                v, search_key, f"{pointer}/{k}" if pointer else f"/{k}"
+            )
+    elif isinstance(input_data, list):
+        for index, item in enumerate(input_data):
+            yield from find_pointers_containing(item, search_key, f"{pointer}/{index}")
+
+
+def resolve_schema(schema_address, language=None):
+    with open(schema_address, "r+") as file:
+        data = json.load(file)
+        pointer_iterator = find_pointers_containing(data, "suggestions_url")
+        for pointer in pointer_iterator:
+            address = resolve_pointer(data, "{}{}".format(pointer, "/suggestions_url"))
+            if language:
+                set_pointer(
+                    data,
+                    "{}{}".format(pointer, "/suggestions_url"),
+                    address.replace("{language_code}", language),
```

Comment:
When a string has `{}`, you can use `.format()`.
So like `address.format(language_code=language)`

Reply:
Added while you were reviewing😀, thanks!